flink-table/flink-table-runtime-blink/src/main/java/org/apache/flink/table/runtime/generated/AggsHandleFunction.java

```diff
@@ -30,9 +30,10 @@
  */
 public interface AggsHandleFunction extends AggsHandleFunctionBase {
 
-	/**
-	 * Gets the result of the aggregation from the current accumulators.
-	 * @return the final result (saved in a row) of the current accumulators.
-	 */
-	RowData getValue() throws Exception;
+    /**
+     * Gets the result of the aggregation from the current accumulators.
+     *
+     * @return the final result (saved in a row) of the current accumulators.
+     */
+    RowData getValue() throws Exception;
```

Comment:
Do I understand it correctly that these classes even though they are located in a package `generated` are actually not generated?

Reply:
Yes. From what I could gather these are used for wrapping/generating code, but not actually generated themselves. The naming is indeed problematic.